scripts/init-rule.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env python3
+# Description:
+# init-rule.py helps you bootstrapping a new rule from scratch.
+# It scaffold and generate code in order to create rules fast.
+#
+# To use:
+# Run the script from the regolibrary base directory in this way:
+# python3 scripts/init-rule.py --name "name-of-new-rule"
+# To get help:
+# python3 scripts/init-rule.py --help
+
+import argparse
+import re
+from os import path
+import os
+import sys
+
+program_name = 'init-rule.py'
+regex = "((([a-zA-Z0-9]+)\-)+([a-zA-Z0-9]+))"
+rules_base_path = 'rules'
+test_list_default = 'success,failed'
+msga = """{{
+    	"alertMessage": "{alert_message}",
+    	"packagename": "armo_builtins",
+    	"alertScore": {alert_score},
+    	"failedPaths": ["{failed_paths}"],
+    	"fixPaths":[{fix_paths}],
+        "fixCommand": "{fix_command}",
+    	"alertObject": {{
+        }}
+    }}
+"""
+raw_rego = """
+package armo_builtins
+
+deny[msga] {{
+
+	msga := {}
+}}
+"""
+rule_metadata = """{{
+    "name": "{rule_name}",
+    "attributes": {{
+      "armoBuiltin": true,
+      "hostSensorRule": {host_sensor_rule},
+      "useFromKubescapeVersion": "{use_from_kubescape_version}",
+      "useUntilKubescapeVersion": "{use_until_kubescape_version}",
+      "imageScanRelated": {image_scan_related},
+      "controlConfigInputs": "{control_config_inputs}"
```

Comment:
IMO we should add these attributes only if we get a value in the args (also - not add attributes with false values which are redundant)

Reply:
Actually, I'd leave them. They could serves as placeholder for future rule improvements. WDYT?